packages/p2p-v2/src/components/PageReturn/PageReturn.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import clsx from 'clsx';
+import ArrowLeftIcon from '../../public/ic-arrow-left.svg';
+import './PageReturn.scss';
+
+type TPageReturnProps = {
+    className?: string;
+    onClick: () => void;
+    pageTitle: string;
+};
+
+const PageReturn = ({ className = '', onClick, pageTitle }: TPageReturnProps) => {
+    return (
+        <div className={clsx('p2p-v2-page-return', className)}>
+            <ArrowLeftIcon className='p2p-v2-page-return__button' onClick={onClick} />
+            <span>{pageTitle}</span>
+        </div>
+    );
+};
+
+export default PageReturn;
```

Comment:
added this component since its used in tandem with other components on desktop such as the verification component, but is not always attached to the specific component.